lib/property/property.controllers.js

```diff
@@ -677,9 +677,7 @@ module.exports = (models) => {
     if (remainder === 0) {
       return true;
     }
-
```

Comment:
while the spaces here aren't needed, it's preferred not to make changes to files that weren't part of your ticket. please revert

Reply:
Fair enough.  I forgot I did that haha.  Tbh, I really wanna rewrite it to just say "return (remainder === 0);"  I'm an adherent of boolean zen, personally speaking lol.